test/IECoreHoudini/SceneCacheTest.py

```diff
@@ -2319,8 +2319,8 @@ def testRopTopLevelGeo( self ) :
 
 		rop.parm( "execute" ).pressButton()
 
-		# unable to write a cache of 3 cortex objects without a name attribute
-		self.assertEqual( len( rop.errors() ) , 1 )
+		# Skips over 3 cortex objects.
+		self.assertEqual( len( rop.errors() ) , 0 )
```

Comment:
I'd have thought there was still an error here since you used `addError` in the writer. Maybe those get reported as warnings instead?

Reply:
This is resolved right as I've changed it back to reporting an error?